scripts/subdetector_tests/draw_bemc_scfi_grids.py

```diff
@@ -1,5 +1,8 @@
 '''
```

Comment:
Can you add an SPDX header here to pass the checks?

Reply:
Add it to the commit https://github.com/eic/epic/pull/370/commits/38c228c9f9f239721080466018646a979781dc43